dev/io.openliberty.org.apache.myfaces.3.0/bnd.bnd

```diff
@@ -0,0 +1,136 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.org.apache.myfaces.3.0
+Implementation-Version: 3.0.0
```

Comment:
Bundle name/description missing when comparing to our 2.3 bundle. We can probably just open an issue to address all the bundle name/description updates we want to make for consistency and to not delay getting the initial drop of this code into a build.